client/src/utils/jobsUtil.js

```diff
@@ -124,6 +136,22 @@ export function getJobDisplayInformation(job, app) {
       //   return true;
       // });
 
+      // Note: Code below also filters v3 parameters by cross referenceing but utilizes more resources
+      // The quicker workaround solution is implemented in Line 63 and 65 and utilizes Array.filter method
+      /*
+      display.parameters.filter((input) => {
+        const matchingParameter = app.definition.jobAttributes.parameterSet.appArgs.find((obj) => {
+          return input.label === obj.name
+        });
+
+        console.log("matching", matchingParameter)
+        if (matchingParameter && matchingParameter.notes.isHidden) {
+          display.parameters.splice(display.parameters.findIndex(item => item.label))
+        }
+        return true
+      })
+      */
+
```

Comment:
I added the comments  Line 139 - 153 since the initial workaround to filter the parameters worked by finding the matchingParameter first and checking to see if it has a `isHidden: true` property on the object. My solution (Line 63-69) just maps through the `parameterSet.appArgs` object array and checks to see if the property `isHidden`. (it doesn't need to find the matchingParameter unlike previous implementation). I wasn't sure of why it was implemented that way so I left it into a comment so I can come back to it if there was a specific design choice and need to implement it this way. The comment Line 104-137 was from before.

Reply:
Removed these comments.